src/audio/lib/thread.c

```diff
@@ -95,7 +95,7 @@ AudioTask* AudioThread_UpdateImpl(void) {
     if (gAudioCtx.resetStatus != 0) {
         if (AudioHeap_ResetStep() == 0) {
             if (gAudioCtx.resetStatus == 0) {
-                osSendMesg(gAudioCtx.audioResetQueueP, gAudioCtx.specId, OS_MESG_NOBLOCK);
+                osSendMesg(gAudioCtx.audioResetQueueP, (OSMesg)(uintptr_t)gAudioCtx.specId, OS_MESG_NOBLOCK);
```

Comment:
Is the double cast needed?

Reply:
Annoyingly casting a non 32-bit wide type to a pointer type produces this warning
```
src/audio/lib/thread.c: In function 'AudioThread_UpdateImpl':
src/audio/lib/thread.c:98:56: error: cast to pointer from integer of different size [-Werror=int-to-pointer-cast]
   98 |                 osSendMesg(gAudioCtx.audioResetQueueP, (OSMesg)gAudioCtx.specId, OS_MESG_NOBLOCK);
      |                                                        ^
```